app/src/main/java/com/simply/birthdayapp/auth/registration/presentation/RegistrationScreen.kt

```diff
@@ -129,9 +203,18 @@ fun RegistrationScreen(viewModel: RegistrationViewModel = koinViewModel()) {
                     text = stringResource(R.string.register),
                     isEnabled = viewModel.isRegisterEnabled.value
                 ) {
-                    // Handle register action
+                    viewModel.signUp(
+                        SignUpInput(
+                            email = email,
+                            firstName = name,
+                            lastName = surname,
+                            password = password
+                        )
+                    )
```

Comment:
you dont need to pass values from screen because you already have in viewmodel so your function in screen can be viewmodel.singup() , keep screen cleaner